game.py

```diff
@@ -16,7 +19,7 @@ def __init__(self, dim=SCREEN_SIZE, fps=30, background_color=(255,255,255)):
 
         self.listeners = defaultdict(list)
 
-    def on(self, element, evt, callback):
+    def on(self, element, evt, callback=None):
```

Comment:
In what scenario would we want to register an element with an event and not have a callback?